src/libraries/System.Private.CoreLib/src/System/Runtime/Loader/LibraryNameVariation.Unix.cs

```diff
@@ -35,31 +35,26 @@ internal static IEnumerable<LibraryNameVariation> DetermineLibraryNameVariations
 
                 bool containsDelim = libName.Contains(Path.DirectorySeparatorChar);
 
-                if (containsSuffix)
+                if (containsSuffix && containsDelim)
```

Comment:
I am not sure whether the reformatting is an improvement. The new code is less compact and I think it makes it harder to see the differences between the different combinations.